### ISSUE TITLE
Invalid use of <br> element

### ISSUE THREAD

**Issue by xydlebug** [xydlebug/X-ttt#7]
The `<br>` element is being used to add vertical spacing to content. These should be replaced by combinations of `<p>`, `<div>` and CSS.

Pages impacted: Home page, Info page.
